### PR TITLE
fix(route): 修复 hellogithub 路由错误

### PR DESCRIPTION
**Pull request by CaoMeiYouRen**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/hellogithub/volume
/hellogithub/month
/hellogithub/ranking
/hellogithub/report
/hellogithub/article
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by CaoMeiYouRen** [comment]
/test
```
/hellogithub/volume
/hellogithub/month
```

File: lib/routes/hellogithub/volume.ts [hunk 1]
```diff
@@ -10,53 +10,69 @@ const md = MarkdownIt({
...
+                    const data = response.props;
+                    const id = data.pageProps.volume.current_num;
+                    return {
+                        title: `《HelloGitHub》第 ${id} 期`,
+                        link: `${rootUrl}/periodical/volume/${id}`,
+                        description: art(path.join(__dirname, 'templates/volume.art'), {
+                            data: data.pageProps.volume.data,
+                        }),
+                    };
+                },
```
Comment (TonyRL) [inline lib/routes/hellogithub/volume.ts:66]: Please handle 404 as well. `https://hellogithub.com/periodical/volume/90` is not available.

File: lib/routes/hellogithub/volume.ts [hunk 2]
```diff
@@ -10,53 +10,69 @@ const md = MarkdownIt({
...
+                config.cache.routeExpire,
+                true
```
Comment (TonyRL) [inline lib/routes/hellogithub/volume.ts:68]:
```suggestion
                false
```

File: lib/routes/hellogithub/volume.ts [hunk 3]
```diff
@@ -10,53 +10,69 @@ const md = MarkdownIt({
...
-    maintainers: ['moke8', 'nczitzk'],
+    example: '/hellogithub/volume',
+    name: '月刊',
+    maintainers: ['moke8', 'nczitzk', 'CaoMeiYouRen'],
     handler,
 };
 
-async function handler() {
+async function handler(ctx) {
+    const limit: number = fallback(queryToInteger(ctx.req.query('limit')), 10);
```
Comment (TonyRL) [inline lib/routes/hellogithub/volume.ts:30]: `fallback` is used to compare between 3 variables. You can use `||` or `??` for 2 variables.

File: lib/routes/hellogithub/volume.ts (hunk 1 above)
Comment (CaoMeiYouRen) [reply lib/routes/hellogithub/volume.ts:66]: > Please handle 404 as well. `https://hellogithub.com/periodical/volume/90` is not available.

My IP can be accessed normally https://hellogithub.com/periodical/volume/90 This issue cannot be replicated locally. Perhaps due to IP restrictions.
我的ip可以正常访问 https://hellogithub.com/periodical/volume/90，本地无法复现这个问题。可能是受ip限制。

**Pull request merged**